File: BCD.Domain/Entities/Pessoa.cs
using System.Collections.Generic;
using BCD.Domain.Entities.Enums;

namespace BCD.Domain.Entities
{
    public class Pessoa
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string CPF { get; set; }
        public EnumEstado Situacao { get; set; }
        public List<Conta> Contas { get; set; }
        public int EnderecoId { get; set; }
        public Endereco Endereco { get; set; }
    }
}

File: BCD-App/src/app/Models/Pessoa.ts
import { Endereco } from './Endereco'
import { Conta } from './Conta'

export class Pessoa {
        id: number;
        nome: string;
        cpf: string;
        // 0 -> Ativada, 1 -> Desativada, 2 -> Bloqueada, 3 -> Analise
        situacao: number;
        contas: Conta[];
        enderecos: Endereco[];
}